docs/40_smart-contract-guides/92_read-only-queries.md

```diff
@@ -0,0 +1,337 @@
+---
+content_title: "Read-Only Queries"
+link_text: "Read-Only Queries"
+---
+
+
+External clients were limited to accessing chain state data through the get_tables APIs for chainbase or KV tables or through various history solutions. In version 2.2 of the there is now an API to invoke contract-defined read-only queries and return the results. Smart contracts can be written to include actions which are read-only queries using the `read-only` attribute. This tutorial will show you how to create a simple smart contract with a read-only query, and how to call this query from the cleos CLI. 
+
+Once the tutorial is completed you should be able to create and call a read-only query.
```

Comment:
Do we always lack confidence in the users?  'should' -> 'will' unless it is specifically our style in other documents to use 'should'.

Reply:
imperative is deemed incorrect/rude and I have been pulled up for using will before.

Reply:
This is the problem with writing instructions in the second person, which I've never liked.